index.html

```diff
@@ -7,6 +7,70 @@
     <title>Music Store App</title>
 </head>
 <body>
-    
+    <header>
+        <h1>
+            <u>Music Store App</u>
+        </h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="">
+                        Instrumente
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        Vinyls
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        Software
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        Zubehör
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        Impressum
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <h2>
+                Herzlich Willkommen im Music Store
+            </h2>
+            <img src="/assets/Music-Logo.jpg" alt="Music Logo" width="300px" />
+        </section>
+        <section>
+            <p>
+                <h2>Anmeldung zum Newsletter</h2>
+            </p>
+            <div>
+                Vorname:
+                <br>
+                <input type="text" />
+                <br>
+                Nachname:
+                <br>
+                <input type="text" />
+                <br>
+                eMail:
+                <br>
+                <input type="email" />
+            </div>
+        </section>
+    </main>
+    <footer>
+        <p>
+            <i>Powered by Patrick Ferrari</i>
```

Comment:
```suggestion
            Powered by Patrick Ferrari
```
again styling in CSS